SMTP_VERIFICATION.md

```
@@ -0,0 +1,152 @@
+
+Expected output:
+
+```
+{"level":"INFO","component":"EmailService","message":"Email transporter initialized",...}
+{"level":"INFO","component":"EmailService","message":"SMTP connection verified successfully"}
+{"level":"INFO","component":"Server","message":"Email service ready"}
+```
+
+### 3. Send Test Email (Admin Panel)
+
+1. Login to admin panel: http://chata.utia.cas.cz/admin.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)